### PR TITLE
CU-86a16b5d5 - NEON3 - Wallet List + Account List

### PR DESCRIPTION
**Pull request by raulduartep**

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a16b5d5 NEON3 - Wallet List + Account List](https://app.clickup.com/t/86a16b5d5)

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a16b5d5 NEON3 - Wallet List + Account List](https://app.clickup.com/t/86a16b5d5)

File: src/renderer/src/components/AccountCard.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      <AccountIcon account={account} />
+
+      <div className="flex justify-between flex-grow">
+        <div className="flex flex-col flex-grow">
+          <p className="text-xs text-gray-100">{account.name}</p>
+          <span className="text-sm text-white">{formattedTotalTokensBalances}</span>
+        </div>
+
+        <div className="flex flex-col justify-between">
+          {/* TODO: REPLACE THE MOCKED DATA WHEN THERE IS A SOLUTION FOR BALANCE VARIATION */}
```
Comment (melanke) [inline src/renderer/src/components/AccountCard.tsx]: Is there a task for this? Put the link or task code on this TODO coomment, please.

File: src/renderer/src/components/WalletCard.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      <WalletIcon wallet={wallet} withAccounts={iconWithAccounts} />
+
+      <div className="flex justify-between flex-grow">
+        <div className="flex flex-col flex-grow">
+          <p className="text-xs text-gray-100">{wallet.name}</p>
+          <span className="text-sm text-white">{formattedTotalTokensBalances}</span>
+        </div>
+
+        <div className="flex flex-col justify-between">
+          {/* TODO: REPLACE THE MOCKED DATA WHEN THERE IS A SOLUTION FOR BALANCE VARIATION */}
```
Comment (melanke) [inline src/renderer/src/components/WalletCard.tsx]: do the same think on all "TODO" comments of the application. This can become a nightmare to maintain.

**Review by melanke — CHANGES_REQUESTED** [review]

**Pull request merged**